src/iac_scan_runner/results_persistence.py

```diff
@@ -0,0 +1,92 @@
+import pymongo
+import bson.json_util as json_util
+from bson.json_util import dumps
+import json
+from datetime import datetime
+
+class ResultsPersistence:
+    def __init__(self):
+        """
+        Initialize new scan result database, collection and client
+        """        
+        self.myclient = pymongo.MongoClient("mongodb://localhost:27017/") 
+        self.mydb = self.myclient["scandb"]
+        self.mycol = self.mydb["results"]
+
+    def parse_json(self, data):
+        return json.loads(json_util.dumps(data))
+
+    def insert_result(self, result: dict):
+        """Inserts new scan result into database
+        :param result: Dictionary holding the scan summary
+        """         
```

Comment:
Start all comments in new row like this (as it's clearer):

```suggestion
        """
        Inserts new scan result into database
        :param result: Dictionary holding the scan summary
        """         
```